src/Core/Services/EnvironmentService.cs

```diff
@@ -125,5 +128,17 @@ private string FormatUrl(string url)
             }
             return url.Trim();
         }
+
+        public async Task<EnvironmentUrlData> SetUsUrlsAsync()
+        {
+            return await SetUrlsAsync(new EnvironmentUrlData()
+            { Base = DefaultBaseUsRegion });
+        }
+
+        public async Task<EnvironmentUrlData> SetEuUrlsAsync()
+        {
+            return await SetUrlsAsync(new EnvironmentUrlData()
+            { Base = DefaultBaseEuRegion });
+        }
```

Comment:
🤔 🎨  Not sure how likely it would be to add more environments but this could be just one method with an enum or create some value objects that can be passed, or my favorite create static `EnvironmentUrlData` like this and remove both methods and use `SetUrlsAsync` directly :
```c#
public class EnvironmentUrlData
{
    public static EnvironmentUrlData DefaultUS = new EnvironmentUrlData { Base = "https://vault.bitwarden.com" };
    public static EnvironmentUrlData DefaultEU = new EnvironmentUrlData { Base = "https://vault.bitwarden.eu" }

    public string Base { get; set; }
    ...
}

// and this can be used like:

await SetUrlsAsync(EnvironmentUrlData.DefaultUS);
await SetUrlsAsync(EnvironmentUrlData.DefaultEU);
```